templates/helm.yml

```diff
@@ -135,16 +135,15 @@
     - &helmapply |
       cat << EOF > /tmp/replacer.sh
       #!/bin/bash
-      sed 's/name: "${NAMESPACE}"/name: "${APP_NAME}"/g' | sed 's/-tls/-tls-${APP_NAME}/g' | awk '/deployment.yaml|service.yaml|ingress.yaml/,/---/'
+      sed 's/name: "${NAMESPACE}"/name: "${APP_NAME}"/g' | sed 's/-tls/-tls-${APP_NAME}/g' | awk '/deployment.yaml|service.yaml|ingress.yaml|configmap.yaml/,/---/'
       EOF
 
       chmod 755 /tmp/replacer.sh
 
-      cat /tmp/replacer.sh
-      
       if [ -n "${APP_NAME_BRANCH}" ]; then
         echo ${APP_NAME}
         echo ${DOMAIN}
+        #helm template "current-repo/${CHART_NAME}" -f /tmp/values.yaml --namespace "${NAMESPACE}" --version ${CHART_VERSION} --post-renderer /tmp/replacer.sh --set ingress.ignoreTest=true --set service.ignoreTest=true
```

Comment:
why comment?

Reply:
This is used form debugging, I don't want template to be printed out in the CI